### ISSUE TITLE
[Bug]: Contextual Help does not read body text to screen readers

### ISSUE THREAD

**Issue by chad-schroeder** [adobe/spectrum-web-components#5382]
### Code of conduct

- [x] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-contextual-help

### Expected behavior

When opening the Contextual Help component, the user should be able to navigate and hear aloud the Heading, Body text and Links.

### Actual behavior

When opening the Contextual Help component, the popover body text is not accessible (Voiceover controls do not navigate). In the React version I'm able to jump to the Heading and/or link nested inside. In both the SWC and React versions I'm unable to hear the body text inside the popover.

### Screenshots

<img width="497" alt="Image" src="https://github.com/user-attachments/assets/79075995-95ec-42fd-bb2a-d587da3198f8" />

### What browsers are you seeing the problem in?

Chrome

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/contextual-help/#example
2. Click on the "!" icon
3. Popover will open.
4. Voiceover should be able to read aloud the body text.
5. See error

### Severity

SEV 2

**Comment by najikahalsema** [comment]
swc-822
